lib/process-request.js

```diff
@@ -6,11 +6,13 @@ const objectToArray = require('./object-to-array');
 
 module.exports = (req, options = {}) => {
   if (options.blacklist) {
-    req.body = removeProperties(req.body, options.blacklist);
+    if (options.blacklist.body) req.body = removeProperties(req.body, options.blacklist.body);
+    if (options.blacklist.headers) req.headers = removeProperties(req.headers, options.blacklist.headers);
   }
 
   if (options.whitelist) {
```

Comment:
In the PHP SDK, I made it so if you supply a blacklist, we don't worry about processing a whitelist: https://github.com/readmeio/metrics-php/blob/master/src/Metrics.php#L185-L189

Should we do the same here?